src/engines/html5/html5.js

```diff
@@ -200,6 +202,23 @@ export default class Html5 extends FakeEventTarget implements IEngine {
     }
   }
 
+  /**
+   * Trigger a playing event whenever an audio track is changed.
+   * This align Edge and IE behaviour to other browsers. When an audio track changed in IE & Edge, they trigger
+   * waiting event but not playing event.
+   * @returns {void}
+   */
+  _handleWaitingEventOnAudioTrack(): void {
+    const affectedBrowsers = ['IE', 'Edge'];
+    if (affectedBrowsers.includes(Env.browser.name)) {
+      this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.AUDIO_TRACK_CHANGED, () => {
+        this._eventManager.listenOnce(this._el, Html5Events.WAITING, () => {
+          this.dispatchEvent(new FakeEvent(Html5Events.PLAYING));
```

Comment:
@odedhutzler  2 issues that we need to fix here:
1. we saw this issue only with hls right? so lets move the code to hls
2. if we change the audio and really going into a buffer the UI will not show it right?